composer.json

```diff
@@ -2,7 +2,10 @@
     "name": "prismic/php-sdk",
     "description": "PHP development kit for Prismic",
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
```

Comment:
Adding `config.platform.php` with a value of "8.0" will prevent composer installing deps for 8.1+ when running on newer PHP versions

Reply:
Do you think this is mendatory? I again checked the [documentation](https://getcomposer.org/doc/06-config.md#platform) and didn't see the need. Could you maybe elaborate on your idea behind this?

If I am missing something obvious please feel free to let me know or add this as a commit 🙂

Reply:
It's helpful to tools like renovate and also means that you can manage deps locally without screwing up the lockfile for the minimum supported version of PHP - i.e. You're working locally on PHP 8.2 and commit a composer update - composer will resolve dependencies as if you were running PHP 8.0 AFAIK

Reply:
So you would suggest adding:
```JSON
"config": {
    "platform": {
        "php": "8.0"
    }
}
```
And then running `composer install` and `composer update` to ensure it's all resolved correctly?

Reply:
Yes, if you are running composer locally via 8.1 or newer, you should find that the lock file doesn't change, assuming it's already up to date for v8.0

Reply:
Locally, I am running `PHP 8.1.2 (cli) (built: Jan 21 2022 04:46:45) (NTS)`. After adding the `config.platform.php` key with a value of `"8.0"` and running `composer install` and `composer update` only the `content-hash`  in the lock file changed. So your assumption seems correct. 

Should I commit the two files?

Reply:
Resolved in Commit [8d788f7](https://github.com/prismicio-community/php-kit/pull/190/commits/8d788f7b034a9f2f79d65ba36b0769bbb970f3a7)